ts/input/tex/ParseMethods.ts

```diff
@@ -29,6 +29,7 @@ import {TexConstant} from './TexConstants.js';
 import {MmlNode} from '../../core/MmlTree/MmlNode.js';
 import ParseUtil from './ParseUtil.js';
 
+const MATHVARIANT = TexConstant.Variant;
```

Comment:
I believe we discussed getting rid of `TexConstant` once these PRs have been merged.

Reply:
Right.